test/Benchmarks/Instrumentation/AspNetCoreInstrumentationBenchmarks.cs

```diff
@@ -76,7 +75,7 @@
 
 Baseline = 2.45 KB
 With Traces and Metrics = Baseline + With Traces + (With Metrics - (Activity creation + `Acitivity.Stop()`)) (they use the same activity)
-                        = 2.45 + (1162 + 64) / 1024 = 2.45 + 1.2 = 3.55 KB (~3.56 KB)
```

Comment:
The previous version seems to have the wrong math 😆

Reply:
Yeah that was a typo but it does match what's in the `Allocated` column in the table (3.66 KB)

= 2.45 + (1162 + 64) / 1024 = 2.45 + 1.2 = **3.65 KB (~3.66 KB)**